.github/ISSUE_TEMPLATE/ui-copy-request.yaml

```
@@ -0,0 +1,97 @@
+      placeholder: |
+        What: We're introducing new feature A.
+        When: This feature will launch at the completion of project B.
+        Why: This feature will make X, Y, and Z easier for the user.
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Related links / assets
+      description: Where can the documentation team learn more about this feature?
+      value: |
+        Please include each of the following, if applicable:
+        Figma link(s):
+        Github epic link(s):
+        Github issue link(s):
+        How to find the text in a production environment:
+        Testing environment information:
+        NOTE: To keep sensitive information such as credentials secure, do not include it in this form.
+    validations:
+      required: true
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which documentation set does this change impact?
+      description: Stateful, Serverless, or both?
+      options:
+        - Stateful and Serverless
+        - Stateful only
+        - Serverless only
+        - Unknown / Not applicable
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: doc-set-differences
+    attributes:
+      label: Feature differences
+      description: If you selected both Stateful and Serverless above, please describe how, if at all, the feature differs in each deployment method.
+      placeholder: The feature is identical in Stateful and Serverless.
+    validations:
+      required: true
+  - type: textarea
+    id: software-version
+    attributes:
+      label: Software version
+      description: Is this work targeted for a particular release version? If so, which one?
+      placeholder: |
+        For example:
+        "8.13.0"; 
+        "has been in production since 8.11.0"; 
+        "N/A"
+    validations:
+      required: true
+  - type: textarea
+    id: collaborators
+    attributes:
+      label: Collaborators
+      description: Please include contact information for the responsible product manager, designer, and developer.
+      value: |
+        PM:
+        Designer: 
+        Developer:
+        Others (if applicable):
+    validations:
+      required: true
+  - type: textarea
+    id: timeline
+    attributes:
+      label: Timeline / deliverables
+      description: When would it be ideal for for us to complete the request? What deliverables do you need, and when? If applicable, how do the release timelines for this feature differ between serverless / stateful? 
+      placeholder: |
+        For example:
+
+        We need a final draft of the new UI copy no later than December 5, ideally by December 1. No differences between serverless / stateful.
+
+        This is an urgent request, it would be great to have updated copy for A and B ASAP, since the flawed copy is already in production. No differences between serverless / stateful.
```

Nit: I don't think most people would call someone else's writing "flawed," maybe "original" or "draft"?
```suggestion
        This is an urgent request, it would be great to have updated copy for A and B ASAP, since the original copy is already in production. No differences between serverless / stateful.
```